config/initializers/stats_admin.rb

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+module Sufia
+  class StatsAdmin
+    def self.matches?(request)
+      current_user = request.env['warden'].user
```

Comment:
Is this specifically getting information on user 'warden'?  Would we need to hardcode all admin users in this way?

Did you see the [Adding and admin user](https://github.com/projecthydra/sufia#adding-an-admin-user) section in the github doc for Sufia?